packages/block-editor/src/utils/dom.js

```diff
@@ -11,6 +11,15 @@ export function getBlockDOMNode( clientId ) {
 	return document.querySelector( '[data-block="' + clientId + '"]' );
 }
 
+export function getClockPreviewContainerDOMNode( clientId ) {
+	const domNode = getBlockDOMNode( clientId );
+	if ( ! domNode ) {
+		return;
+	}
+
+	return domNode.querySelector( '.wp-block-preview-container' );
```

Comment:
You're relying on this selector being available on all Blocks. How can we guarantee that will be the case? Are we planning to add this selector to all Core Blocks? What about 3rd party Blocks.

This is the central issue at play here which I referenced in the original PR, namely that you cannot accurately and consistently target an element which wraps the _visual_ content of a Block.

I'm really interested to know if @youknowriad sees a way in which we could target consistently and what he feels about the approach outlined here.

Reply:
I've resolved this now. We now just looked a the `firstChild`. It's not perfect but it's the best we can do.